### ISSUE TITLE
Canon EOS 90D gets stuck in Busy after gphoto2 config commands

### ISSUE THREAD

**Issue by ylf-3b** [gphoto/gphoto2#465]
**Describe the bug**
The gphoto2 commands to get or set configs over USB let the camera stuck in busy with open PC connection.
gphoto summary for example doesn't have this behaviour. 
When I remember right, we already have had this behaviour several years ago, in cause of a fault in the canon PTP driver.
In opposite my old Canon 350D, witch runs with the older canon driver, have no problems with that. So this might be a bug?

**Name the camera**
Modell                         Port                                            
----------------------------------------------------------
Canon EOS 90D                  usb:001,006     

**libgphoto2 and gphoto2 version**
gphoto2         2.5.27         gcc, popt(m), exif, no cdk, no aa, jpeg, readline
libgphoto2      2.5.27         standard camlibs, gcc, ltdl, EXIF
libgphoto2_port 0.12.0         iolibs: disk ptpip serial usb1 usbdiskdirect usbscsi, gcc, ltdl, EXIF, USB, serial lockdev locking

**To Reproduce**
gphoto2 --summary (everything went well)
gphoto2 --set-config iso=100 (stuck in busy)
gphoto2 --get-config iso (works but still stuck in busy)
gphoto2 --summary (everything is allright now)

**Comment by msmeissn** [comment]
busy being the camera display, right?

**Comment by ylf-3b** [comment]
Yes, the LCD Screen shows the "Remote Connection" Symbol (a PC) and when pressing any button the shoulder display quit's it with the busy message. I think it's feature, that the camera can't be used directly in remote mode.

**Comment by elstoc** [comment]
Same as issues I've raised with the 760D and now experienced on the 80D
